### PR TITLE
update slides

### PR DESCRIPTION
**Pull request by martinfleis**
This slightly updates slides (@jGaboardi included and a different CSS) and moves them here. I also enabled GH Pages so we can serve it directly from this repo.

@jGaboardi If you want to review this, open this branch locally and find `intro.html`.

### CODE REVIEW

**Comment by martinfleis** [comment]
I will replace the link to the slides in the notebook once this gets merged and we're sure it is building and is served.

**Comment by martinfleis** [comment]
We may also want to add something to slides but I am not sure it is necessary.

File: slides/intro.html [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+                </CENTER>
+                <br />
+                <CENTER>
+                <span class="fragment"> <a href="https://pysal.org">pysal.org</a> </span>
+                </CENTER>
+                </section>
+                <section id="momepy" class="slide level2">
+                <h2>momepy</h2>
+                <CENTER class="fragment">
+                <span style="color:#3b6e8c"> Urban Morphology Measuring Toolkit </span> <small> Member of a PySAL family </small>
```
Comment (jGaboardi) [inline slides/intro.html]: ~~Member of a PySAL family~~

--> A member of the PySAL family

**Review by jGaboardi — COMMENTED** [review]
I think this is good to merge after the one edit on the `momepy` slide. Nice work.

**Pull request merged**